platforms/documentation/docs/src/docs/userguide/dep-man/dependency-management/component_metadata_rules.adoc

```diff
@@ -78,153 +72,130 @@ include::sample[dir="snippets/dependencyManagement/customizingResolution-metadat
-For these modules you should only use component metadata rules if you have clearly identified a problem with the metadata itself.
-If you have an issue with the dependency resolution result, you should first check if you can solve the issue by declaring <<rich_versions.adoc#rich-version-constraints,dependency constraints with rich versions>>.
-In particular, if you are developing a library that you publish, you should remember that dependency constraints, in contrast to component metadata rules, are published as part of the metadata of your own library.
-So with dependency constraints, you automatically share the solution of dependency resolution issues with your consumers, while component metadata rules are only applied to your own build.
+Before using component metadata rules to adjust a module’s metadata, determine whether the module was published with **Gradle Module Metadata** (`.module` file) or **traditional metadata** (`.pom` or `ivy.xml`):
 
-If a module was published with traditional metadata (`.pom` or `ivy.xml` only, no `.module` file) it is more likely that the metadata is incomplete as features such as variants or dependency constraints are not supported in these formats.
-Still, conceptually such modules can contain different variants or might have dependency constraints they just omitted (or wrongly defined as dependencies).
-In the next sections, we explore a number existing oss modules with such incomplete metadata and the rules for adding the missing metadata information.
+- **Modules with Gradle Module Metadata**: These typically have complete metadata, but issues can still occur. Only apply component metadata rules if you've clearly identified a problem with the metadata. For dependency resolution issues, first consider using [dependency constraints with rich versions](rich_versions.adoc#rich-version-constraints). If you're developing a library, note that dependency constraints are published as part of your own library’s metadata, making it easier to share the solution with consumers. In contrast, component metadata rules apply only within your own build.
+- **Modules with Traditional Metadata** (`.pom` or `ivy.xml`): These are more likely to have incomplete metadata since features like variants and dependency constraints aren’t supported in these formats. Such modules might have variants or constraints that were omitted or incorrectly defined as dependencies. In the following sections, we explore examples of OSS modules with incomplete metadata and the rules to add missing information.
 
-As a rule of thumb, you should contemplate if the rule you are writing also works out of context of your build.
-That is, does the rule still produce a correct and useful result if applied in any other build that uses the module(s) it affects?
+TIP: Ensure that the rule you create works beyond your build and still provides a correct and useful result if applied in other builds that use the affected module(s).
```

Comment:
💅 Why?  Rules aren't exported or serialized in any way, you'd have to cut-and-paste the code into the new build.  I like the original phrasing here better, and don't think it deserves callout status.

Reply:
Reverted.